### PR TITLE
Add query button entities to ISY994 devices and hub

### PR DESCRIPTION
**Pull request by shbatm**
## Proposed change

Deprecate the `isy994.system_query` service in favor of Query button entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85337
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/isy994.markdown [hunk 1]
```diff
@@ -231,9 +234,11 @@ Send an ISY set_ramp_rate command to a `light` Node to set the devices' ramp rat
 | `entity_id`            |     no   | Name(s) of target entities for the command, e.g., `light.front_porch`. |
 | `value` | no | The integer index value to set the Ramp Rate to in a range of `0` (9.5 minutes) to `31` (0.1 Seconds), e.g., `28` |
 
-#### Service `isy994.system_query`
+#### Service `isy994.system_query` (Deprecated)
```
Comment (frenck) [inline source/_integrations/isy994.markdown]: Deprecated services should be removed from the documentation

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/isy994.markdown (hunk 1 above)
Comment (shbatm) [reply source/_integrations/isy994.markdown]: Updated.

**Review by frenck — APPROVED** [review]
Thanks, @shbatm 👍

**Review by frenck — CHANGES_REQUESTED** [review]
@shbatm A wild merge conflict appeared out of nowhere! 🙈 
Could you take a look?

Thanks! 👍 

../Frenck

**Comment by shbatm** [comment]
My fault--removing several services over a few different PRs.

**Review by frenck — APPROVED** [review]
Thanks, @shbatm 👍 

../Frenck

**Pull request merged**